apps/demos/Demos/Charts/ErrorBars/description.md

```diff
@@ -1,2 +1,13 @@
-If&nbsp;your data contains an&nbsp;error or&nbsp;measurement uncertainty, you can indicate a&nbsp;confidential interval on&nbsp;the Chart using error bars. To&nbsp;configure them, use the fields of&nbsp;the series&rsquo; **valueErrorBar** object. In&nbsp;this example, error bars indicate a&nbsp;range of&nbsp;average temperatures (on&nbsp;the upper pane) and humidity (on&nbsp;the lower pane) for a&nbsp;specific month.
-<!--split-->
\ No newline at end of file
+DevExtreme Chart supports value error bars. Implement these bars to indicate data measurement tolerances and [confidence intervals](https://en.wikipedia.org/wiki/Confidence_interval). Configure error bar settings and appearance in the **series**.[valueErrorBar](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/valueErrorBar/) object.
```

Comment:
DevExtreme Chart supports value error bars. These bars indicate data measurement tolerances and [confidence intervals](https://en.wikipedia.org/wiki/Confidence_interval). Use the **series**.[valueErrorBar](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/valueErrorBar/) object to configure error bar settings and appearance.